characters.py

```diff
@@ -128,6 +143,44 @@ def heal(self, other):
         """
         min(other.current_hp + self.healing_power, other.max_hp)
 
+
+class Voodoo(Character, MagicType):
+    def __init__(self, poison_nova, name):
+        MagicType.__init__(self)
+        Character.__init__(self, 1000, name)
+        self.poison_nova = poison_nova
+
+    def act(self, other):
+        self.poison(other)
+        self.remove_if_dead(other)
+
+    def poison(self, other):
+        for modifier in other.modifier_list:
+            if not isinstance(modifier, modifiers.Poison):
+                if self.lvl > 10:
+                    modifier.duration += int(1 + self.lvl / 2)
+                elif self.lvl > 5:
+                    modifier.duration += int(1 + self.lvl / 3)
+                else:
+                    modifier.duration += 1
+            else:
+                poison = modifiers.Poison(self, other, 3, self.poison_nova)
+                other.modifier_list.append(poison)
+
+    # def poison(self, other):
```

Comment:
A to co to?

Reply:
a co z tym jest nie tak? to bylo od poczatku chyba w ten sposob, nie pamietam, zebym to zmienial

Reply:
zmienilem, teraz dobrze?